projects/plugins/jetpack/extensions/shared/external-media/media-service/index.ts

```diff
@@ -1,3 +1,4 @@
+import jetpackAnalytics from '@automattic/jetpack-analytics';
```

Comment:
In the Jetpack extensions bundle, we've typically relied on the `useAnalytics` hook instead:

```suggestion
import { useAnalytics } from '@automattic/jetpack-shared-extension-utils';
```

Reply:
We can't use hooks in this location, we aren't in React context.